tool/requote.js

```diff
@@ -49,12 +59,17 @@ function findSubstring(sequence, substring, spansBySentencePos) {
 }
 
 
-function findSpanType(program, begin_index, end_index) {
+function findSpanType(program, begin_index, end_index, requote_numbers) {
     let spanType;
     if (begin_index > 1 && program[begin_index-2] === 'location:') {
         spanType = 'LOCATION';
-    } else if (NUMBER_MATCH_REGEX.test(program[begin_index])){
-        spanType = 'NUMBER';
+
+    } else if (do_replace_numbers(program[begin_index], requote_numbers) && !(program[end_index+1].startsWith('^^'))){
+        // catch purely numeric postal code
+        if (program[begin_index - 3].endsWith('String'))
```

Comment:
NO. **Do not** assume that type annotations are present, and **do not** assume that there is a parameter nearby.

You know from the syntax if something is a number or a string. That's the whole point of `"` marks!